README.Rmd

```diff
@@ -15,13 +15,19 @@ knitr::opts_chunk$set(
 )
 ```
 
-# ghqc
+# ghqc <a href="https://github.com/a2-ai/ghqc/"><img src="man/figures/logo.png" align="right" height="139" alt="ghqc website" /></a>
 
 <!-- badges: start -->
-[![Lifecycle: experimental](https://img.shields.io/badge/lifecycle-experimental-orange.svg)](https://lifecycle.r-lib.org/articles/stages.html#experimental)
+[![R-CMD-check](https://github.com/A2-ai/ghqc/actions/workflows/R-CMD-check.yaml/badge.svg)](https://github.com/A2-ai/ghqc/actions/workflows/R-CMD-check.yaml)
 <!-- badges: end -->
 
-The goal of ghqc is to ...
+The goal of the ghqc ecosystem is to simplify, standardize, and improve traceability of the QC process through the use of shiny apps which create GitHub Issues and Milestones for your project. The ghqc package is a lightweight wrapper to be installed along side project packages and has three primary functionalities:
+
+- **Isolating Package Dependency**: Installs, or symlinks, the high dependency burden of the ghqc shiny app ecosystem to an isolated directory to ensure no interference with any project work.
+
+- **Launching Shiny Apps**: Launches the ghqc shiny apps as background jobs to allow for continued use of the R Console
+
+- **Verifying Configuring Information**: Aids in the installation and structure verification of the organization specific configuring information, such as checklists and logos.
```

Comment:
At the end of this section, we can say something like:
"See the README of ghqc.app for a high-level overview of the core shiny app functionality" and link it

Reply:
https://github.com/A2-ai/ghqc/blob/b22c559d5c7ec223217c29fb4be56437a933fce8/README.Rmd#L129

This has been addressed at the bottom. Reference to the standalone website will be added here once it is live